### PR TITLE
Initial setup for the Crash Reporting Shopify app

### PR DESCRIPTION
**Pull request by JasenPalmer**
This is the initial setup of the Crash Reporting app.

I have created a project called `crash-reporting` and an extension within that project called `Raygun Crash Reporting`. 

This is the most basic version of the app. It can be installed in a shop and will report errors to the configured application. 

To run, execute the command 'yarn dev' from within the `\crash-reporting` directory and follow the prompts for setup

### CODE REVIEW

File: crash-reporting/README.md
Comment (crystalqdm) [inline crash-reporting/README.md:1]: We can probably remove this file

File: crash-reporting/SECURITY.md
Comment (crystalqdm) [inline crash-reporting/SECURITY.md:1]: We can probably remove this file

File: crash-reporting/shopify.app.raygun-crash-reporting.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Learn more about configuring your app at https://shopify.dev/docs/apps/tools/cli/configuration
+
+name = "Raygun Crash Reporting"
+client_id = "242392a59ff87b609c4027eee44772c7"
+application_url = "https://shopify.dev/apps/default-app-home"
+embedded = false
```
Comment (crystalqdm) [inline crash-reporting/shopify.app.raygun-crash-reporting.toml]: `embedded = false` is in here twice?

**Review by crystalqdm — APPROVED** [review]
Looks good to me - just non blocking coments

File: crash-reporting/shopify.app.raygun-crash-reporting.toml (hunk 1 above)
Comment (JasenPalmer) [reply crash-reporting/shopify.app.raygun-crash-reporting.toml]: The first embedded is for an embedded app and the second is for a [POS](https://shopify.dev/docs/apps/pos/getting-started) app. I believe I can remove the POS one as it's not required

File: crash-reporting/README.md
Comment (JasenPalmer) [reply crash-reporting/README.md:1]: I have updated the file to be more useful

**Pull request merged**